scout/server/blueprints/institutes/templates/overview/clinvar_submissions.html

```diff
@@ -22,8 +22,8 @@
 <div class="container-float">
   <h3>ClinVar submissions page<h5></h3><a data-target="#howto" href="#" data-toggle="modal">howto</a></h5><br>
   <!-- Modal -->
-  <div class="modal fade w-100" role="dialog" id="howto" aria-hidden="true" tabindex="-1">
-    <div class="modal-dialog" role="document">
+  <div class="modal fade bd-example-modal-lg" role="dialog" id="howto" aria-hidden="true" tabindex="-1">
```

Comment:
`bd-example-modal-lg`?

Reply:
That would be the data target of the example I think, will fix!

Reply:
Fixed, and made the modal extra large:

![image](https://user-images.githubusercontent.com/28093618/112443845-b2ff3500-8d4d-11eb-893c-663f13aa5873.png)